### DISCUSSION TITLE
Why now is not free?

### DISCUSSION THREAD

**Discussion by jighen555** [thunderclient/thunder-client-support#1365] [General]
Hello Ranga,

we use your tool in one of the most automotive company in the world, coming from Germany.
All the areas of the company now is using Thunderclient, but if you put this limitation i'm not sure that the company will continue to use your tool. They are seriously thinking to change the HTTP client, just for your information.

Let me know if it's possible to return into a previous free version, or update the number of calls into a collection, because we have a lot of collections with more than 15 calls unfortunately.

I wait your news,
Thanks!

**Comment by rangav** [comment]
Hi @jighen555 

companies are not allowed to use the free version.

free terms
https://www.thunderclient.com/terms#free-plan

Please purchase a subscription to use premium version and get better support from us.
